spec/store/index-spec.js

```diff
@@ -307,13 +307,15 @@ describe("Store", () => {
       expect(store.notebook).toEqual(commutable.toJS(nb));
     });
 
-    xit("should return a fully-fledged notebook when the file isn't empty", () => {
```

Comment:
The test still doesn't pass, but I'm not sure why because it seems to produce the correct output when I paste the test commands into the dev console.

Reply:
You seem to have discovered why it was failing and made it pass, though It looks like we are testing that there is an empty cell at the top now, which seems like a bug. I know you pointed this out before, is it something to fix?

Reply:
I updated that in #1505 https://github.com/nteract/hydrogen/pull/1505/files